### ISSUE TITLE
Easy delete and copy/paste frames

### ISSUE THREAD

**Issue by ggarra13** [ggarra13/mrv2#94]
### Discussed in https://github.com/ggarra13/mrv2/discussions/82

- Add a delete frame (with sound) option/hotkey.
Quicktime professional player got that built in option and it's so far the only player I've found having that very, very useful possibility even rv player doesn't have. For a professional animator it is a real game changer. If I would have only one request it would be this one.

- Add a copy/paste frame option/hotkey.
Basically a copy/paste frame. Again, for animator use it's great.

**Comment by ggarra13** [comment]
This will be part of the next big update to mrv2 (ETA unknown for now).  You will be able to cut, copy, paste and insert frames to your heart's content.  You will also be able to Undo and Redo any edits.

**Comment by ggarra13** [comment]
This feature is now added in v0.7.5.  It works fine with one clip, but it currently does not work properly if copying frames from one clip to another.

**Comment by ggarra13** [comment]
v0.7.6 fixes a memory corruption issue and makes it possible to copy frames from one video to another.
